src/main/java/seedu/address/model/issue/IssueList.java

```diff
@@ -10,10 +10,11 @@
 import javafx.collections.ObservableList;
 import seedu.address.model.issue.exceptions.DuplicateIssueException;
 import seedu.address.model.issue.exceptions.IssueNotFoundException;
+import seedu.address.model.room.Room;
 
 /**
  * A list of issues that does not allow nulls.
- *
+ * <p>
```

Comment:
What's this tag for?

Reply:
It is auto-added by Intellij as part of the normal way of providing paragraph breaks in javadocs. It is conformant to the javadoc standards: https://www.oracle.com/sg/technical-resources/articles/java/javadoc-tool.html